src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj

```diff
@@ -9,6 +9,13 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
```

Comment:
@jaredmoo So by default every test project get's 1.1.0 reference of Resource manager.
So unless you explicitly add 1.6.0 to the test project you are still using 1.1.0 resource manager nuget
Here is the link for the change
https://github.com/Azure/azure-sdk-for-net/pull/3272/files#diff-c8ac009921fcb73848451401f0d039f2R16